### ISSUE TITLE
Level sync downgrade suggestions

### ISSUE THREAD

**Issue by kaedys** [daemitus/XIVComboPlugin#49]
Been playing a couple jobs that have some automated level-sync downgrades of some abilities, and it's _craaaazy_ useful (I hate the button pattern disruption that comes with synced content on some jobs).  Below are some suggestions for similar graceful downgrades in synced content.

All of them are intended to be direct replacements, ie. when an ability is absent, what would be the other or equivalent ability you'd naturally hit in that situation instead.  I also didn't bother if the only relevant equivalent was only a couple levels lower (ex. Abyssal Drain at level 56 vs Carve and Spit at 60), or if the replacement level threshold was less than 30, since this is mostly meant for people at or near max level that have developed strong muscle memory for their job abilities that sync into low level content via the roulettes.

* [BLM] Xenoglossy -> Foul under level 80
* [BLM] Fire IV -> Fire 1 and Blizzard IV -> Blizzard I when below levels 60 and 58, respectively.
* [GNB] Gashing Fang -> Burst Strike when under level 60 (possibly Double Down as well).
* [PLD] Goring Blade -> Rage of Halone under level 54
* [WAR] Storm's Eye -> Storm's Path under level 50
* [DRK] Edge of Shadow -> Flood of Shadow under level 40
* [MNK] Dragon Kick -> Bootshine under level 50
* [MNK] AoE abilities with single target ones under the level that you gain the AoE.  The most relevant of these is Four-point Fury -> Twin Snakes when under level 45.
* [MNK] Enlightenment -> Forbidden Chakra/Howling Fist when under level 70.
* [SAM] Senei -> Shinten and Guren -> Kyuten under levels 72 and 70, respectively.  However, this may not be needed, since there's already a Shinten -> Senei and Guren -> Kyuten replacement when they're off CD.
* [AST] Aspected Helios -> Helios under level 42
* [WHM] Medica 2 -> Medica under level 50 (possibly also Cure III -> Medica under 40, but Cure III requires more direct attention anyway)
* [DRG] Chaos Thrust -> Full Thrust under level 50

**Comment by MKhayle** [comment]
hi! this issue is _very_ old and probably no longer up to date

I'll try to remember to check every instance of this happening but if I don't do it by the end of july please feel free to comment back here and politely kill me

**Comment by kaedys** [comment]
Oh ya, this one's very out of date now.  Now that I'm more comfortable with the mod code, I'll probably see about implementing this one myself, when I find time.

**Comment by lhn1703** [comment]
Level synced BLM AoE rotation from #323 is a good candidate here. Black mage is the pinnacle of level sync drastically altering the rotation.

**Comment by kaedys** [comment]
I was mostly just intended to do direct 1:1 swaps with this feature, identical to the existing Benefic/Benefic II combo for AST.  Features that handle lower-level compatibility for other features, or more complicated interactions, won't be part of this one.

**Comment by kaedys** [comment]
I don't believe Xenoglossy -> Foul has been implemented.  This is an issue to track the sync features I'd _like_ to implement, not the ones that _have_ been implemented.
